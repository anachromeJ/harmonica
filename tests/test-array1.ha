int main() {
	list[int] l;
	int a=4;
	l[a] = 1;

	printi(l[0]);

}
