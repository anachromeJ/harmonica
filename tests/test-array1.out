999
1
5
